<?xml version="1.0"?>

<launch>
    <arg name="world_file" value="$(find local_planner_benchmark)/worlds/mazes/old_maze/old_maze.world"/>
    <arg name="world_name" value="old_maze"/>
    <arg name="map_file" value="$(find mir_navigation)/map/gazebo_full_map.yaml"/>
    <arg name="dynamic" default="false"/>
    <arg name="move_arm" default="false"/>
    <arg name="initial_x" value="0.0"/>
    <arg name="initial_y" value="0.0"/>
    <arg name="initial_z" value="0.0"/>
    <arg name="initial_yaw" value="0"/>
    <arg name="verbose" value="false"/>
    <arg name="trial_arg" default="1"/>
    <arg name="local_planner" default="dwb"/>

    <group if="$(arg move_arm)">
        <include file="$(find realistic_dora_moveit_config)/launch/move_group.launch"/>
        <node name="get_joint_states" pkg="local_planner_benchmark" type="ee_smoothness_dynamic.py"/>
    </group>

    <include file="$(find local_planner_benchmark)/launch/common_benchmarking.launch" pass_all_args="true">
        <param name="world_name_py" value="$(arg world_name)"/>
    </include>

</launch>